lib/pbench/test/unit/agent/task/test_copy_result_tb.py

```diff
@@ -48,13 +48,13 @@ def test_bad_tar(self, caplog):
             f"http://pbench.example.com/api/v1/upload/{bad_tarball}",
             status=HTTPStatus.OK,
         )
-        expected_error_message = (
-            f"FileNotFoundError: Tar ball '{bad_tarball}' does not exist"
-        )
+        expected_error_message = f"Tar ball '{bad_tarball}' does not exist"
         caplog.set_level(logging.ERROR, logger=self.logger.name)
-        with pytest.raises(FileNotFoundError) as e:
+        with pytest.raises(FileNotFoundError) as excinfo:
             crt = CopyResultTb(
                 "controller", bad_tarball, 0, "ignoremd5", self.config, self.logger
             )
             crt.copy_result_tb("token")
-        assert str(e).endswith(expected_error_message)
+        assert str(excinfo.value).endswith(
```

Comment:
Is this because the message actually changed? 😕

Reply:
The way `pytest.raises` works in the later versions of `pytest` (6.2.5 and later) that is compatible with Python 3.10.

Reply:
Actually, I guess I highlighted the wrong line; I understand the `ExcInfo` change, but I meant to call attention to dropping the `FileNotFoundError: ` prefix on the message. I did that badly, didn't I? 😆

Reply:
Wait; you're saying that under Python 3.10 `str(FileNotFoundError)` doesn't include the "FileNotFoundError: " prefix anymore? Wow.

Reply:
No, I think worse, the new version of Pytest seems to strip it, because this works under Python 3.6 thru 3.10.